.github/workflows/build_docs.yml

```
@@ -21,14 +20,9 @@ jobs:
       - name: Install common packages
         uses: ./.github/actions/install-dependencies
 
-      - name: Install book deps
-        run:
-          python3 -m pip install -r requirements.txt
-
       - name: Build the book
         run: 
-          jupyter-book build . -W
-
+           conda run -n mpi-tutorial jupyter-book build . -W
```

Oops, it's called `mpi` (or rename the env above)

```suggestion
           conda run -n mpi jupyter-book build . -W
```

---

I've been changing it back and forth, as  it seems like there has to be an env naming consistency.

---

Sorry about that.